### ISSUE TITLE
Users

### ISSUE THREAD

**Issue by AntonioCS** [slashfan/LexikJWTAuthenticationBundleSandbox#3]
Hey,

I am trying to understand where exactly the user is suppose to come from.
I can see that everything works but I am still a bit unclear on how to properly use this.

I have my own api (not using FOSRestBundle) and I want to use this but where do I add the users and passwords? Or do I need to use something like FOSUserbundle for that?

**Comment by AntonioCS** [comment]
Sorry, I forgot that Symfony handles all of this via the security thing. 
I saw the users there. I assume the /api/login_check is just to tell where the user should go so that symfony internally runs whatever code it needs to run to identity the user. Am I wrong?

**Comment by sketchthat** [comment]
Correct, the `api/login_check` is where Symfony runs the logic to identify the user. It is defined in `app/config/security.yml` Line 25

**Comment by mappedinn** [comment]
Should we create an action for the route `api/login_check`?
